modules/ServerAPI/src/client/SaltBrowser.html

```diff
@@ -99,8 +99,8 @@ <h5>Searching for Salts containing text "<i class="bv_exptSearchTerm"></i>"</h5>
         </div>
         <div class="well well-small bv_saltControllerContainer hide" style="margin-top: 10px;">
             <div>
-                <button class="btn btn-danger bv_deleteSalt">Delete</button>
-                <button class="btn bv_editSalt pull-right">Edit</button>
+                <button class="btn btn-danger hide bv_deleteSalt">Delete</button>
```

Comment:
Hid these by default, so it simplifies the logic to only show them to admins. Also fixes a tiny UX issue where they'd flash visible then hidden again for CmpdReg Users